CIAC-13029: Pull All Incidents (#39692)
demisto/content: Packs/IronscalesEventCollector/ReleaseNotes/1_1_0.md

#### Integrations

##### Ironscales Event Collector

> - Added new **Collect all incidents** parameter, upon checking it all incidents will be fetched instead of open ones only.

Reviewer: ```suggestion
Added the new **Collect all incidents** parameter. When checked, it fetches all issues instead of just open ones.
```
incidents?alerts?issues?events?
